src/index.html

```diff
@@ -4,9 +4,31 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" type="image/png" href="/favicon.ico"/>
```

Comment:
Totally forgot to add these! Good thinking. (These don't require dynamic injection like the one listed above, obviously.)